### PR TITLE
Changelog for v0.13  

### PR DESCRIPTION
**Pull request by LukeMathWalker**
I have filled in the changelog for the next release (3 pages of PRs :sweat_smile:).
It should be quick enough to update it with the very last PRs before releasing.

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -4,22 +4,80 @@ Version 0.13.0 (in development)
...
- - Implement approx traits for `ArrayBase`
-   ([#581](https://github.com/rust-ndarray/ndarray/pull/581) by @jturner314)
+   ([#617](https://github.com/rust-ndarray/ndarray/pull/617) by [@JP-Ellis])
+ - Implement approx traits for `ArrayBase`. They can be enabled using the `approx` feature-flag.
+   ([#581](https://github.com/rust-ndarray/ndarray/pull/581) by [@jturner314])
+ - Add `mean` method
+   ([#580](https://github.com/rust-ndarray/ndarray/pull/580) by [@LukeMathWalker])
+ - Add `Zip::all` to check if all elements of an iterator satisfy a predicate
+   ([#615](https://github.com/rust-ndarray/ndarray/pull/615) by [@mneumann])
+ - Add `CoWArray`, `C`lone `o`n `Write` array
```
Comment (bluss) [inline RELEASES.md]: I guess this is CowArray

**Comment by bluss** [comment]
Awesome, thanks for working on it

File: RELEASES.md [hunk 2]
```diff
@@ -4,22 +4,80 @@ Version 0.13.0 (in development)
...
+ - Add `nth_back` method to `ndarray`'s iterators
+   ([#686](https://github.com/rust-ndarray/ndarray/pull/686) by [@jturner314]) 
+ - Add `split_at` method to `AxisChunksIter/Mut`
+   ([#691](https://github.com/rust-ndarray/ndarray/pull/691) by [@jturner314])
+ - Add `into_scalar` method to `ArrayView0` and `ArrayViewMut0`
+   ([#700](https://github.com/rust-ndarray/ndarray/pull/700) by [@LukeMathWalker])
 
 Enhancements
 ------------
+ - Improve performance for matrix multiplications when using the pure-Rust backend thanks to `matrix-multiply:v0.2`
```
[the remark this answers is not in the document]
Comment (LukeMathWalker) [reply RELEASES.md]: Updated - I think the change scope is clearer now

**Comment by max-sixty** [comment]
Thanks for the cred! 

I'll try and do some more work on ndarray—really appreciate all the guidance you guys have offered @LukeMathWalker @jturner314 (and @bluss for the library)

I have a branch for the Named Dimensions. It's not exactly seamless in rust but I'll push and get feedback after the next round of work

**Pull request merged**